resource-timing/worklet-rt-entries.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8" />
+<title>Resource Timing Entry Sequence of Events for Worklets</title>
+<link rel="help" href="https://w3c.github.io/resource-timing/"/>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<script>
+    promise_test(async() => {
+        const url = new URL("/worklets/resources/empty-worklet-script.js", location.href).toString();
+        await CSS.paintWorklet.addModule(url);
```

Comment:
Are there other worklet types we should be testing? I'm not sure which browsers support which worklets, tbh, but [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Worklet) mentions
- PaintWorklet
- AudioWorklet
- AnimationWorklet
- LayoutWorklet

Reply:
Yea I don't think they're really supported, I wanted to have a go with one type of the most common worklet to get started

Reply:
Sticking to PaintWorklet for now SGTM!